sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -7,10 +7,14 @@ import {
 import HttpBaseClient from "./common/HttpBaseClient";
 
 export class ConfirmationOfEnrollmentApi extends HttpBaseClient {
-  public async getCOESummary(locationId: number): Promise<COESummaryDTO[]> {
+  public async getCOESummary(
+    locationId: number,
+    upcomingCOE?: boolean,
+  ): Promise<COESummaryDTO[]> {
     try {
+      const enrollmentPeriod = upcomingCOE ? "upcoming" : "current";
```

Comment:
how about creating an enum for upcomingCOE with `"upcoming" ,  "current"`

Reply:
Enum used now.